package.json

```diff
@@ -614,6 +614,13 @@
           "group": "navigation@1"
         }
       ],
+      "explorer/context": [
+        {
+          "command": "SAS.convertNotebookToFlow",
+          "when": "resourceExtname == .sasnb && SAS.connectionType == rest",
```

Comment:
It's not supported in `serverId` case, right?
`resourceExtname == .sasnb && !SAS.connection.direct && SAS.connectionType == rest`

Reply:
You're right. Updated in https://github.com/sassoftware/vscode-sas-extension/pull/552/commits/87f18fa61f9ad249a625be6ecdb403b9b3394532